worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/BukkitEntity.java

```diff
@@ -89,7 +89,7 @@ public boolean setLocation(Location location) {
 
     @Override
     public com.sk89q.worldedit.world.entity.EntityType getType() {
-        return EntityTypes.get(type.getName().toUpperCase(Locale.ROOT));
```

Comment:
apparently this isn't allowed to be upper case

Reply:
`[20:41:29] [Craft Scheduler Thread - 4/ERROR]: An unexpected error while handling a WorldEdit command
java.lang.IllegalStateException: key must be lowercase
	at com.google.common.base.Preconditions.checkState(Preconditions.java:444) ~[patched_1.15.2.jar:git-Paper-366]
	at com.sk89q.worldedit.registry.Registry.get(Registry.java:53) ~[?:?]
	at com.sk89q.worldedit.registry.NamespacedRegistry.get(NamespacedRegistry.java:55) ~[?:?]
	at com.sk89q.worldedit.world.entity.EntityTypes.get(EntityTypes.java:135) ~[?:?]
	at com.sk89q.worldedit.bukkit.BukkitEntity.getType(BukkitEntity.java:92) ~[?:?]
	at com.sk89q.worldedit.function.operation.ForwardExtentCopy.lambda$resume$2(ForwardExtentCopy.java:389) ~[?:?]`